modules/openapi-generator/src/test/resources/3_0/petstore.yaml

```diff
@@ -701,7 +701,7 @@ components:
             wrapped: true
           items:
             type: string
-        tags:
+        tags: 
```

Comment:
What about removing this trailing space as well?

Reply:
Done and pushed.